src/shared/ui/Toast/Toast.stories.tsx

```
@@ -0,0 +1,93 @@
+      showToast({
+        type: "success",
+        message: "성공적으로 처리되었습니다!",
+      }),
+    );
+
+    return WithProvider(() => <Toast />);
+  },
+};
+
+export const Error: Story = {
+  render: () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)